<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectTaskWizard</class>
 <widget class="QWizard" name="SelectTaskWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select an option</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="importRadioButton">
      <property name="text">
       <string>Import setting from file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="createNewRadioButton">
      <property name="text">
       <string>Create a new setting</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="installRadioButton">
      <property name="text">
       <string>Install one setting that is already created</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Setting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="settingComboBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2"/>
 </widget>
 <resources/>
 <connections/>
</ui>
